.github/workflows/gsExample.yaml

```diff
@@ -9,6 +9,9 @@ on:
     branches: [ main, master ]
   pull_request:
     branches: [ main, master ]
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1"
```

Comment:
Maybe add some random hour and minute in the middle of the night to relieve stress on build servers?